config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Looks like this can be deleted rather than commenting out the whole file, if we're not going to use it...